tests/Nox.Reference.GetTestData/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿Console.WriteLine("Starting Nox.Reference Data collection...");
+Console.WriteLine();
+
+var path = new DirectoryInfo(Directory.GetCurrentDirectory());
+
+while (!Directory.Exists(Path.Combine(path.FullName, ".git")))
```

Comment:
Can we use In-Memory configuration for UT

Reply:
Added a todo, will solve if all is fine after Andre approves